src/dotnet/APIView/APIView/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("csharp-api-parser, PublicKey=0024000004800000940000" +
```

Comment:
Why are you exposing the internals here?

Reply:
I need to use CodeAnalysis apis https://github.com/Azure/azure-sdk-tools/blob/54008272ebdfa7606d0126bd24a5739db3d9b060/src/dotnet/APIView/APIView/SymbolDisplay/AbstractSymbolDisplayVisitor.cs#L10

Reply:
We should look at either exposing or sharing source for this instead of exposing internal apis.